src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/CustomTimestampSerialiser.java

```diff
@@ -5,29 +5,16 @@
 import com.fasterxml.jackson.databind.ser.std.StdSerializer;
 
 import java.io.IOException;
-import java.sql.Timestamp;
 import java.time.Instant;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
 
-import static java.time.ZoneOffset.UTC;
-
-public class CustomTimestampSerialiser extends StdSerializer<Timestamp> {
+public class CustomTimestampSerialiser extends StdSerializer<Instant> {
```

Comment:
Like with the deserialiser, worth changing the name to sth like `CustomInstantSerialiser` or even `InstantSerialiser`, as the very fact of having it in our code implies that it's custom.